frontend/www/js/omegaup/contest/edit.ts

```diff
@@ -65,12 +65,29 @@ OmegaUp.on('ready', () => {
           })
           .catch(ui.apiError);
       },
-      refreshProblems: (): void => {
+      refreshProblems: (problemAdded: boolean): void => {
         api.Contest.problems({
           contest_alias: payload.details.alias,
         })
           .then((response) => {
             contestEdit.problems = response.problems;
+            if (problemAdded) {
+              if (
+                contestEdit.problems[contestEdit.problems.length - 1].languages
+                  .split(',')
+                  .includes('cat') &&
+                !payload.details.languages.includes('cat')
```

Comment:
```suggestion
                !contestEdit.details.languages.includes('cat')
```

oops, creo que se me fue esto en el comentario de antes.